readthedocs/oauth/services/gitlab.py

```diff
@@ -602,7 +602,7 @@ def send_build_status(self, build, commit, state):
                 'GitLab commit status creation failed.',
                 debug_data=debug_data,
             )
-        except CustomOAuth2Error as e:
+        except OAuth2Error as e:
```

Comment:
Why not catching directly `InvalidGrantError` and removing the following `if`?

Reply:
That makes more sense, but I figured there was a reason it was implemented this way 🤷

Reply:
Nope, the only reason was that I didn't find the correct/exact exception as you did 😄